psql2mysql.py

```diff
@@ -61,6 +61,37 @@ def _exclude_deleted(self, table, query):
             return query.where(table.c.deleted == 'False')
         return query.where(table.c.deleted == False)
 
+    def getTextColumns(self, table):
+        columns = table.columns
+        return [ c.name for c in columns if str(c.type) == 'TEXT' ]
+
+    def scanTableForLongTexts(self, table):
+        textColumns = self.getTextColumns(table)
+        if not textColumns:
+          return []
+        LOG.debug("Scanning Table %s (columns: %s) for too long TEXT values ",
+                 table.name, textColumns)
+        filters = [
+            text("length(\"%s\") > %i" % (x, MAX_TEXT_LEN)) for x in textColumns
+        ]
+        q = table.select().where(or_(f for f in filters))
+        rows = q.execute()
+
+        long_values = []
+        primary_keys = []
+        if not table.primary_key == None:
```

Comment:
`if table.primary_key is not None` or `if table.primary_key`. Both are equivalent.

Reply:
Oh, how could I missed this